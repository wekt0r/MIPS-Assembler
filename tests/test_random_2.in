xori $23, $27, 9978
slt $0, $9, $27
sltu $28, $5, $23
div $s3, $24
srl $s2, $s6, 6
sllv $13, $s1, $a2
div $a0, $s6
srav $4, $4, $k0
srlv $8, $27, $3
srl $t3, $18, 6
slt $4, $v0, $s4
sub $t2, $29, $a3
addu $fp, $t1, $26
srl $a3, $a2, 23
sltiu $zero, $6, 23402
slt $29, $26, $t4
sub $at, $23, $s1
mfhi $16
mfhi $27
lui $k0, 2685
multu $a1, $19
divu $23, $5
subu $at, $v1, $a2
multu $v1, $8
xor $s1, $20, $24
div $19, $14
sltiu $8, $sp, 11069
addiu $19, $6, 781
addu $24, $s0, $t8
mult $t9, $at
sllv $ra, $a2, $sp
mult $18, $10
xor $28, $31, $20
andi $gp, $21, 17933
subu $17, $29, $8
srl $k1, $24, 19
nor $13, $9, $26
addi $13, $10, 26594
andi $v0, $a1, 22991
srav $t8, $fp, $a3
ori $6, $21, 471
or $gp, $4, $28
srav $t5, $28, $s3
mfhi $t6
sllv $1, $6, $t4
xori $8, $ra, 4071
addiu $6, $v0, 29475
xori $21, $a0, 16662
ori $s5, $31, 23071
sll $27, $gp, 7
sub $12, $16, $11
slt $s2, $29, $t2
and $22, $8, $24
divu $26, $9
andi $t5, $s2, 21236
sllv $16, $k1, $s4
or $20, $s5, $t5
or $sp, $14, $16
addu $6, $fp, $27
srlv $15, $30, $t9
xor $28, $4, $21
