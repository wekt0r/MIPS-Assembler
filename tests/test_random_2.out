.text
00000000	3B7726FA	xori	$23,$27,9978
00000004	013B002A	slt	$0,$9,$27
00000008	00B7E02B	sltu	$28,$5,$23
0000000c	0278001A	div	$s3,$24
00000010	00169182	srl	$s2,$s6,6
00000014	00D16804	sllv	$13,$s1,$a2
00000018	0096001A	div	$a0,$s6
0000001c	03442007	srav	$4,$4,$k0
00000020	007B4006	srlv	$8,$27,$3
00000024	00125982	srl	$t3,$18,6
00000028	0054202A	slt	$4,$v0,$s4
0000002c	03A75022	sub	$t2,$29,$a3
00000030	013AF021	addu	$fp,$t1,$26
00000034	00063DC2	srl	$a3,$a2,23
00000038	2CC05B6A	sltiu	$zero,$6,23402
0000003c	034CE82A	slt	$29,$26,$t4
00000040	02F10822	sub	$at,$23,$s1
00000044	00008010	mfhi	$16
00000048	0000D810	mfhi	$27
0000004c	3C1A0A7D	lui	$k0,2685
00000050	00B30019	multu	$a1,$19
00000054	02E5001B	divu	$23,$5
00000058	00660823	subu	$at,$v1,$a2
0000005c	00680019	multu	$v1,$8
00000060	02988826	xor	$s1,$20,$24
00000064	026E001A	div	$19,$14
00000068	2FA82B3D	sltiu	$8,$sp,11069
0000006c	24D3030D	addiu	$19,$6,781
00000070	0218C021	addu	$24,$s0,$t8
00000074	03210018	mult	$t9,$at
00000078	03A6F804	sllv	$ra,$a2,$sp
0000007c	024A0018	mult	$18,$10
00000080	03F4E026	xor	$28,$31,$20
00000084	32BC460D	andi	$gp,$21,17933
00000088	03A88823	subu	$17,$29,$8
0000008c	0018DCC2	srl	$k1,$24,19
00000090	013A6827	nor	$13,$9,$26
00000094	214D67E2	addi	$13,$10,26594
00000098	30A259CF	andi	$v0,$a1,22991
0000009c	00FEC007	srav	$t8,$fp,$a3
000000a0	36A601D7	ori	$6,$21,471
000000a4	009CE025	or	$gp,$4,$28
000000a8	027C6807	srav	$t5,$28,$s3
000000ac	00007010	mfhi	$t6
000000b0	01860804	sllv	$1,$6,$t4
000000b4	3BE80FE7	xori	$8,$ra,4071
000000b8	24467323	addiu	$6,$v0,29475
000000bc	38954116	xori	$21,$a0,16662
000000c0	37F55A1F	ori	$s5,$31,23071
000000c4	001CD9C0	sll	$27,$gp,7
000000c8	020B6022	sub	$12,$16,$11
000000cc	03AA902A	slt	$s2,$29,$t2
000000d0	0118B024	and	$22,$8,$24
000000d4	0349001B	divu	$26,$9
000000d8	324D52F4	andi	$t5,$s2,21236
000000dc	029B8004	sllv	$16,$k1,$s4
000000e0	02ADA025	or	$20,$s5,$t5
000000e4	01D0E825	or	$sp,$14,$16
000000e8	03DB3021	addu	$6,$fp,$27
000000ec	033E7806	srlv	$15,$30,$t9
000000f0	0095E026	xor	$28,$4,$21
